### DISCUSSION TITLE
Search not working

### DISCUSSION THREAD

**Discussion by drummingman77** [Budibase/budibase#9083] [Help]
Here's my project URl: https://ikestest.budibase.app/app/ikestest

I was trying to connect a search to my table using this tutorial: https://docs.budibase.com/docs/adding-a-search-field#1-using-the-table-block-the-easy-way

However, when I go click "Define Filters" and go to select my text input field that I'm using for search (called Property Search), I am not seeing it. 

What am I doing wrong? I have the text input field nested inside of the data provider in the components view, so I'm not sure why I can't get the search to function.

**Comment by melohagan** [comment]
Hey @drummingman77 

Could you please share screenshots of your component tree and your filter. 

What data type is Property Search?

**Comment by drummingman77** [comment]
Sorry for the late reply!

I'm not sure what you mean by data type. I used a form element 'text input'
for the search. Let me know what other screen shots you need, and I
apologize in advance. I'm new to all of this.

**Comment by drummingman77** [comment]
Here's the filter screenshot attached:

**Reply by melohagan** [reply]
I don't see a screenshot

**Comment by drummingman77** [comment]
Ok, I'll include them in the body this time:
[image: filters.PNG]
[image: search 1.PNG]
